Dockerfile

```diff
@@ -1,9 +1,33 @@
-FROM dhub.ncats.dhs.gov:5001/cyhy-core-with-latex-geos
-MAINTAINER David Redmin <david.redmin@hq.dhs.gov>
+FROM cisagov/cyhy-core as cyhy-core-with-latex-geos
```

Comment:
I think `as` should be `AS` here, since it is a keyword for Dockerfiles.

Reply:
Thanks- changed in https://github.com/cisagov/cyhy-reports/pull/57/commits/3de52c16055b958e58d958d20b66f352da115fd5.